bibliopixel/util/threads/compose_events.py

```diff
@@ -2,10 +2,12 @@
 
 
 def compose_events(events, condition=all):
-    """Compose a sequence of events into one event.
-    Arguments:
-        events:    a sequence of objects looking like threading.Event
-        condition: a function taking a sequence of bools and returning a bool.
+    """compose_events(events, condition=all)
```

Comment:
ALL docstrings are to define function params as shown here. Any that exist with the old format should be changed. All new ones must follow this pattern. See here for more: http://www.sphinx-doc.org/en/stable/domains.html#info-field-lists

example:
```
:param str sender: The person sending the message
   :param str recipient: The recipient of the message
   :param str message_body: The body of the message
   :param priority: The priority of the message, can be a number 1-5
   :type priority: integer or None
   :return: the message id
   :rtype: int
   :raises ValueError: if the message_body exceeds 160 characters
   :raises TypeError: if the message_body is not a basestring
```

Reply:
I don't quite understand this first half of the comment.  More, shouldn't `all` be in quotes?

Reply:
Also, we lose information in this edit and in the next one too...!

Reply:
No... because all is literally the python built-in function`all` - If you look in the code, it's not a string. It's just that function object.
And yeah... I removed the return value details. Will add back.

Reply:
Wait... what is getting lost here? I see nothing that didn't make it into the new stuff.